pkg/cmd/root.go

```diff
@@ -47,8 +47,14 @@ func init() {
 	RootCmd.PersistentFlags().BoolVar(&disableDockerIntegration,
 		"disable-docker-integration", false, "Do not attempt to list/stop/start"+
 			" docker containers which are using dvol volumes")
+}
+
+func initConfig() {
+	if _, err := os.Stat(basePath); err != nil {
+		os.MkdirAll(basePath, 0600)
+	}
 
 	viper.SetConfigName("config")
-	viper.AddConfigPath("$HOME/.dvol")
+	viper.AddConfigPath(basePath)
 	viper.ReadInConfig()
```

Comment:
I'm thinking just mandate it with `GetConfigFile` as we explicitly only write to one file using the CLI.